deployability/modules/workflow_engine/examples/dtt1-agents-poc-aws.yaml

```diff
@@ -0,0 +1,106 @@
+version: 0.1
+description: This workflow is used to test agents deployment por DDT1 PoC
+variables:
+  agent-os:
+    - linux-ubuntu-20.04-amd64
+    - linux-ubuntu-22.04-amd64
+    - linux-oracle-9-amd64
+    - linux-amazon-2-amd64
+
+    #- linux-redhat-7-amd64
+    - linux-redhat-8-amd64
+    - linux-redhat-9-amd64
+    - linux-centos-7-amd64
+
+    - linux-centos-8-amd64
+    - linux-debian-10-amd64
+    - linux-debian-11-amd64
+    - linux-debian-12-amd64
+
+    - linux-amazon-2023-amd64
+    - linux-suse-15-amd64
+    #- linux-opensuse-15-amd64
+
+  manager-os:
+    #- linux-oracle-9-amd64
+    #- linux-ubuntu-20.04-amd64
+    #- linux-centos-8-amd64
+    - linux-redhat-7-amd64
```

Comment:
Should those lines be commented on?

Reply:
Done in [6311d92](https://github.com/wazuh/wazuh-qa/pull/5190/commits/6311d92724994e505494d41ca00dd7d9cddda76f)